docs/graphene-workload-tutorial/create_workload.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python3
+
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+from jinja2 import Template
+
+if len(sys.argv) != 2:
+    print("Name of workload required as argument.")
+    sys.exit(1)
+
+name = sys.argv[1]
+
+# Normalize name to lowercase
+workload = name.lower()
+
+# Ask user for name of workload
+print("Name of new workload: {}".format(workload))
+
+# Convert to get the name of the class 
+workload_classname = workload.title()
+
+# Template files to be processed 
+FILES = {
+    "Dockerfile",
+    "compose/graphene-sgx.yaml",
+    "docker-compose.yaml",
+    "graphene_sgx/build_gsc_workload.sh",
+    "src/my_workload.py",
+    "tests/test_work_orders.json",
+    "workload.json"
+}
+
+# Switch working directory to workload
+os.chdir("./workload")
```

Comment:
Can we directly create the output files at "./workload" directory instead of changing the directory

Reply:
Created a separate directory altogether instead of updating the existing files. The new files are organized in a directory named same as the workload. So, it needs only to be moved and no new directory creation required.